parser/Node.h

```diff
@@ -40,7 +50,7 @@ template <class To> To *cast_node(Node *what) {
     static_assert(!std::is_pointer<To>::value, "To has to be a pointer");
     static_assert(std::is_assignable<Node *&, To *>::value, "Ill Formed To, has to be a subclass of Expression");
 #if __cplusplus >= 201402L
-    static_assert(std::is_final<To>::value, "To is not final");
+    // static_assert(std::is_final<To>::value, "To is not final");
```

Comment:
I had to temporarily disable this, since the classes we're checking against are no longer `final` :/